src/classes/STG_InstallScript_TEST.cls

```diff
@@ -504,9 +504,10 @@ public with sharing class STG_InstallScript_TEST {
                         new BDI_MigrationMappingHelperMock());
         BDI_MigrationMappingUtility util= new BDI_MigrationMappingUtility(mock_BDI_MigrationMappingHelper);
 
-        Test.testInstall(new STG_InstallScript(util), new Version(1,0), true);
-        System.assertEquals(2, util.queuedMetadataTypesForDeploy.size(),
-                'We have 2 metadata records queued for deployment.');
+        Test.testInstall(new STG_InstallScript(util), new Version(1,0), true);     
+        // we should have 2 field mappings queued for deployment   
+        System.assertEquals(2, util.queuedMetadataTypesForDeploy.size(), 
+            'We should have 2 mapping metadata records queued for deployment.');
```

Comment:
Is this the spot you're referring to "fix unit test in STG_InstallScript_TEST?"

Is it feasible to modify the stub class to include a new default object + new default field so we can see if both the object and the field are migrated? There are a few different stubs in BDI_MigrationMappingUtility_TEST that we might be able to reference.

Reply:
I tried that last-night, looks like a really big effort and I couldn't make it work. It's not just mocking some fields/object, we need to mock an entire BDI_MigrationMappingHelper class (including the methods). 
I also tried modifying the existent ones but it breaks a lot of unit tests so I end it up doing the same as you did previously, just re-used what is there.